### DISCUSSION TITLE
Seeed Studio XIAO ESP32C6 によるピンの多重化 | Seeed スタジオ Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1119] [Q&A]
# Seeed Studio XIAO ESP32C6 によるピンの多重化 | Seeed スタジオ Wiki

Pin multiplexing with Seeed Studio XIAO ESP32C6.

https://wiki.seeedstudio.com/xiao_pin_multiplexing_esp33c6/

**Comment by msfujino** [comment]
Where can I find the schematic?
